source/client-side-encryption/client-side-encryption.rst

```diff
@@ -413,21 +413,23 @@ example:
 
    class AutoEncryptionOpts {
       // setTLSOptions accepts a map of KMS provider names to TLSOptions.
-      // Throws an exception if the KMS provider name is not "kmip".
+      // Throws an exception if TLS options are set for a provider other
```

Comment:
Consider removing this.  What's the harm in configuring TLSOptions for "local"?  We can just say something generic like: The options will be used to configure any socket required to communicate with the KMS.

Reply:
The harm is ignoring TLS options on misconfiguration. But I do not think it poses a security or correctness risk, since TLS is enabled by default.

Will remove.